theme/templates/accounts/profile.html

```diff
@@ -415,7 +414,7 @@ <h3 class="panel-title">
                                             {#        State       #}
                                             <fieldset>
                                                 <label>State/Province *</label>
-                                                <input name="state" class="form-control form-required" {% if  profile.state %}value="{{ profile.state }}"{% else %}value="Unspecified"{% endif %}>
+                                                <input name="state" class="form-control form-required" value="{{ profile.state }}">
```

Comment:
Should country be added to?

Reply:
@sblack-usu I have made a new commit to address this. Basically, for those existing users who have not specified country and state, they should be left empty (rather than None or unspecified) in the input fields on the profile page so that they have to select valid values in order to save updated profile. This will make sure no unspecified country or None state got saved to user profile any more. You can test it out on the new-playground.hydroshare.org.